README.md

```diff
@@ -424,9 +424,9 @@ This class is defined as part of the ElementaryFunctionLibrary
 
 The Page reference for Signal has a Unique identifier that can be used as Name.
 
-The AML reader application should find the same Page reference Name in the DocumentReference - AML file and then the signal path can be followed at the opposite drawing.
+The AML reader application should find the same Page reference Name in the DrawingReference - AML file and then the signal path can be followed at the opposite drawing.
 
-![](illustrations/image066.png) ![](illustrations/image068.png)
+![](illustrations/image066.png) ![](illustrations/image068_New.png)
```

Comment:
Please trust git to track changes and provide access to old versions of content, also for binary files.
The old image should be deleted, not kept as floating garbage.
Because this is a binary file, the filename and content should not both change at the same time, eg, within a single commit (git traceability would be broken), which - if squash merging is used for the PR - means the filename and content should not both change within the same PR.